src/Templates/src/templates/maui-mobile/MauiApp.1.csproj

```diff
@@ -19,8 +19,6 @@
 		<ImplicitUsings>enable</ImplicitUsings>
 		<Nullable>enable</Nullable>
 <!--#if (IncludeSampleContent) -->
-		<!-- https://github.com/CommunityToolkit/Maui/issues/2921 -->
-		<NoWarn>NU1608</NoWarn>
```

Comment:
This also needs to be removed in the tests stuff.

https://github.com/dotnet/maui/blob/c5df3b655336494a498530ccfb2f8778bf64646e/src/TestUtils/src/Microsoft.Maui.IntegrationTests/SimpleTemplateTest.cs#L49